.github/workflows/lint.yml

```diff
@@ -21,12 +21,13 @@ jobs:
           pip install --upgrade pip
           pip install flake8
           # stop the build if there are Python syntax errors or undefined names
-          flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+          flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics --exclude=.git,__pycache__,data,demo,jars,.idea,venv,.tox,.parquet,.json.gz,.csv
           # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-          flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+          flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics --exclude=.git,__pycache__,data,demo,jars,.idea,venv,.tox,.parquet,.json.gz,.csv
```

Comment:
Is this necessary when we have a `.flake8`? Or a flake8 section in the `pyproject.toml`?

Reply:
both shall works, honestly didn't check pyproject.toml. It can goes also under setup.cfg

Reply:
At this point, I'm fine with putting it in one of these places, but we should not exclude folders in the workflow as well. They should be defined in one place only :-)